dali/operators/reader/loader/video/frames_decoder.h

```diff
@@ -70,6 +71,23 @@ struct AvState {
   }
 };
 
+/**
+ * @brief Helper representing video file kept in memory. Allows reading and seeking.
+ * 
+ */
+struct MemoryVideoFile {
+  explicit MemoryVideoFile(char *data, int64_t size)
+    : data_(data), size_(size), position_(0) {}
+
+  int Read(unsigned char *buffer, int buffer_size);
+
+  int64_t Seek(int64_t new_position, int origin);
```

Comment:
Any short docu explaining what is the second argument?

Reply:
Done.